### ISSUE TITLE
Jitpack.io build failures

### ISSUE THREAD

**Issue by jtang-ml** [bumptech/glide#3877]
**Glide Version**: 4.7.1

**Stack trace / LogCat**:
```ruby
https://jitpack.io/com/github/bumptech/glide/v4.7.1/build.log

Build starting...
Start: Wed Apr 18 06:34:30 UTC 2018
Git:
v4.7.1
commit 6ea67b6b3e634759c8b81c2c5f26487709c4c267
Author: Sam Judd 
Date:   Fri Apr 13 11:05:15 2018 -0700

    Bump version to 4.7.1


Run gradle build
Gradle build script
Found gradle version: 4.6.
Using gradle wrapper
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Djavax.xml.accessExternalSchema=all
Downloading https://services.gradle.org/distributions/gradle-4.6-all.zip
.....................................................................................................
Unzipping /home/jitpack/.gradle/wrapper/dists/gradle-4.6-all/bcst21l2brirad8k2ben1letg/gradle-4.6-all.zip to /home/jitpack/.gradle/wrapper/dists/gradle-4.6-all/bcst21l2brirad8k2ben1letg
Set executable permissions for: /home/jitpack/.gradle/wrapper/dists/gradle-4.6-all/bcst21l2brirad8k2ben1letg/gradle-4.6/bin/gradle

------------------------------------------------------------
Gradle 4.6
------------------------------------------------------------

Build time:   2018-02-28 13:36:36 UTC
Revision:     8fa6ce7945b640e6168488e4417f9bb96e4ab46c

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
JVM:          1.8.0_131 (Oracle Corporation 25.131-b11)
OS:           Linux 4.14.15-xxxx-std-ipv6-64 amd64

0m4.640s
Getting a list of gradle tasks
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Djavax.xml.accessExternalSchema=all
fatal: Not a git repository: /home/jitpack/v0.9/work1/../gitdir/com/github/bumptech/glide/.git/modules/library/test/src/test/resources/exif-orientation-examples
Unable to find current revision in submodule path 'library/test/src/test/resources/exif-orientation-examples'
Found javadoc task
Running: ./gradlew clean -Pgroup=com.github.bumptech -Pversion=v4.7.1 -xtest -xlint install
Picked up JAVA_TOOL_OPTIONS: -Dfile.encoding=UTF-8 -Djavax.xml.accessExternalSchema=all
fatal: Not a git repository: /home/jitpack/v0.9/work1/../gitdir/com/github/bumptech/glide/.git/modules/library/test/src/test/resources/exif-orientation-examples
Unable to find current revision in submodule path 'library/test/src/test/resources/exif-orientation-examples'
Configuration on demand is an incubating feature.

FAILURE: Build failed with an exception.

* Where:
Build file '/home/jitpack/build/integration/okhttp/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':integration:okhttp'.
> Failed to apply plugin [id 'com.android.library']
   > Your project contains 2 or more modules with the same identification com.github.bumptech:test
     at ":library:test" and ":annotation:compiler:test".
     You must use different identification (either name or group) for each modules.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
EXIT_CODE=1
2018-04-18T06:34:55.697693608Z
Exit code: 1
```

**Comment by jtang-ml** [comment]
Indeed I have checked maven repo the builds fine. Looks like the it's not just a particular release issue at jitpack. Check this out:
https://jitpack.io/#bumptech/glide

All of the builds have some sort of issues. Majority of them pointing to 
`Your project contains 2 or more modules with the same identification com.github.bumptech:test
     at ":library:test" and ":annotation:compiler:test".
     You must use different identification (either name or group) for each modules.`

I got around this for now as a temporary solution by reconfiguring gradle repositories priorities.
